Firmware/firmwareLiBCM/config.h

```diff
@@ -151,6 +151,7 @@
     #define LIDISPLAY_CELL_COLOR_BIN_SIZE_COUNTS 64 //64 = 6.4mV window between cell colours on the grid charging page.  Don't go below CELL_BALANCE_TO_WITHIN_COUNTS_LOOSE
```

Comment:
Move these three constants into LiDisplay.h
A typical user isn't going to interact with these values.

Reply:
I concede that I have several lingering debug parameters in this same location... I need to move them elsewhere, but haven't done so yet... please do this now, and I will move mine later.